streampark-common/src/main/scala/org/apache/streampark/common/util/PropertiesUtils.scala

```diff
@@ -101,6 +101,12 @@ object PropertiesUtils extends Logger {
     }
   }
 
+  def fromYamlTextToJava(text: String): java.util.Map[String, String] = {
```

Comment:
There is already a method: [fromYamlTextAsJava](https://github.com/apache/incubator-streampark/blob/9c7387fcce3a19c6074d0eeb9d0979409c8529af/streampark-common/src/main/scala/org/apache/streampark/common/util/PropertiesUtils.scala#L212)